packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -29,27 +29,77 @@ Vertical navigation is hierarchical global navigation that displays navigation o
 * You have secondary navigation items (even if you have less than five primary navigation items)
 * You expect your application to be used on mobile devices
 
-#### Variations
+### Verical navigation variants
```

Comment:
It feels weird to me so have "Vertical navigation variants" as an H3, on the same level as the "Vertical navigation" header. I wonder if we should just get rid of the "Usage" header, and move each header below it up a level, so that Vertical Nav, Horizontal nav are H2s, and then the actual variations below are H3. So maybe even getting rid of the "Vertical nav variants" header so that the variations themselves can be the header that appears in the table of contents.

For clarity, this is what I would expect after those changes:

- Elements
- Vertical Navigation (and variants)
- - - Simple vertical nav
- - - Grouped navigation
- - - Expandable-two level nav
etc


> 



<img width="352" alt="image" src="https://user-images.githubusercontent.com/51165119/158619674-2d7dddf4-674f-4a3a-b692-ca75014b32d6.png">